boot/freeldr/bootsect/extldr.S

```diff
@@ -0,0 +1,279 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     EXTFS second stage loader
+ * COPYRIGHT:   Copyright 2024 Daniel Victor <ilauncherdeveloper@gmail.com>
+ */
+
+#define INCLUDED_ASM
+#include "ext.S"
+
+ExtLdrEntryPoint:
```

Comment:
Which parts of this file (I think, which functions) are being used in the case of EXT4, compared to the case of EXT2/3 ?